docs/install/install-cave.md

```diff
@@ -127,16 +133,41 @@ To run CAVE, either:
 
 ### System Requirements
 
+- Will need admin privileges to install python package
 - NVIDIA Graphics card is recommended, some Intel Graphics cards will working
 
 > Note: Most AMD graphics cards are not supported
 
 ### Download and Installation Instructions
-1. Download and install both: [**awips-cave.dmg** <i class="fa fa-download"></i>](https://www.unidata.ucar.edu/downloads/awips2/awips-cave.dmg) and [**awips-python.pkg** <i class="fa fa-download"></i>](https://www.unidata.ucar.edu/downloads/awips2/awips-python.pkg)
```

Comment:
I honestly don't know, but it seems like you would want to install python first, then CAVE like on Windows, but maybe it's not necessary?

Reply:
This does not need to happen.
This is the order I just installed on my mac, and was able to then run cave and show wind barbs and precip type as a test (products that use jep on the client side).